pkg/private/install.py.tpl

```diff
@@ -103,15 +103,27 @@ class NativeInstaller(object):
 
     def _install_treeartifact(self, entry):
         logging.info("COPYTREE %s <- %s/**", entry.dest, entry.src)
-        raise NotImplementedError("treeartifact installation not yet supported")
-        for root, dirs, files in os.walk(entry.src):
-            relative_installdir = os.path.join(entry.dest, root)
+        shutil.copytree(
+            src = entry.src,
+            dst = entry.dest,
+            symlinks = True,
+            copy_function=lambda s, d: self._do_file_copy(s, d, entry.mode, entry.user, entry.group),
+            dirs_exist_ok = True,
+        )
+
+        dir_mode = None
```

Comment:
Can you add a comment that this is cleaning up the owners of the directories because copytree will create them as the owner running install.

That *is* what this is doing, right?